data/nabadeeptalukdar.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Nabadeep Talukdar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I'm a frontend developer"
+  "avatar": "https://github.com/nabadeeptalukdar.png"
```

Comment:
In the recent update, we don't need to add avatar: anymore, it will auto-pick your GitHub profile picture. So, please remove that. If you need any help feel free to ask.

Updated docs: https://linkfree.eddiehub.io/docs/quickstart